src/python/grpcio/build.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2016, Google Inc.
```

Comment:
How 'bout I slap a `TODO` on there to replace the hack with a less ugly hack (e.g. dissecting the command for `spawn`) and we call it an unblocking-windows-testing-day?